bucket/keyferret.json

```
@@ -0,0 +1,16 @@
+{
+    "version": "3.0",
+    "description": "Type accents, symbols, maths, Cyrillic, Greek, Hebrew and IPA in any Windows application",
+    "homepage": "https://www.keyferret.com/",
+    "license": "MIT",
+    "url": "https://www.keyferret.com/download/7/",
```

```suggestion
    "url": "https://www.keyferret.com/download/7/#/dl.zip",
```
This renames the downloaded file to `dl.zip`, to prevent errors caused by redirections.